src/js/wrapper.start.js

```diff
@@ -11,7 +11,21 @@
     define(['jquery'], factory);
   } else if (typeof exports === 'object') {
     // Node/CommonJS
-    factory(require('jquery'));
+    module.exports = function (root, $) {
+      if (!root) {
+        // CommonJS environments without a window global must pass a
+        // root. This will give an error otherwise
+        root = window;
+      }
+
+      if (!$) {
+        $ = typeof window !== 'undefined' ? // jQuery's factory checks for a global window
```

Comment:
I prefer the [UMD patterns template](https://github.com/umdjs/umd/blob/95563fd6b46f06bda0af143ff67292e7f6ede6b7/templates/jqueryPlugin.js#L10-L21) with how it breaks this ternary conditional out into a full conditional block.

```
if (jQuery === undefined) {
  if (typeof window !== 'undefined') {
    jQuery = require('jquery');
  } else {
    jQuery = require('jquery')(root);
  }
}
```